File: Android/RapiMoncha/src/una/android/rapimoncha/entidades/Usuario.java
package una.android.rapimoncha.entidades;


import org.json.JSONArray;
import org.json.JSONObject;

import una.android.rapimoncha.interfaces.IJsonable;

import java.text.DateFormat;
import java.text.SimpleDateFormat;
import java.util.Calendar;

/**
 * Created by Lester on 31/03/2015.
 */
public class Usuario implements IJsonable {
    private int idUsuario;
    private String noUsuario;
    private String a1Usuario;
    private String a2Usuario;
    private Calendar feNacimien;
    private String seUsuario;
    private String username;
    private String password;
    public String fechanacString;

    public Usuario() {
    }

    public int getIdUsuario() {
        return idUsuario;
    }

    public void setIdUsuario(int idUsuario) {
        this.idUsuario = idUsuario;
    }

    public String getNoUsuario() {
        return noUsuario;
    }
    public String getDateTimeFeNacimiento(){
        /* DateFormat dateFormat = new SimpleDateFormat("yyyy-MM-dd");
         return dateFormat.format(feNacimien.getTime());*/
    	return fechanacString;
    }

    public void setNoUsuario(String noUsuario) {
        this.noUsuario = noUsuario;
    }

    public String getA1Usuario() {
        return a1Usuario;
    }

    public void setA1Usuario(String a1Usuario) {
        this.a1Usuario = a1Usuario;
    }

    public String getA2Usuario() {
        return a2Usuario;
    }

    public void setA2Usuario(String a2Usuario) {
        this.a2Usuario = a2Usuario;
    }

    public Calendar getFeNacimien() {
        return feNacimien;
    }

    public void setFeNacimien(Calendar feNacimien) {
        this.feNacimien = feNacimien;
    }

    public String getSeUsuario() {
        return seUsuario;
    }

    public void setSeUsuario(String seUsuario) {
        this.seUsuario = seUsuario;
    }

    @Override
    public JSONObject getJson() {
        JSONObject usuariosub=new JSONObject();
        JSONObject finale=new JSONObject();
       try {

           usuariosub.put("idusuario", getIdUsuario());
           usuariosub.put("nousuario", getNoUsuario());
           usuariosub.put("a1usuario", getA1Usuario());
           usuariosub.put("a2usuario", getA2Usuario());
           usuariosub.put("fenacimiento", getDateTimeFeNacimiento());
           usuariosub.put("sexo", getSeUsuario());
           usuariosub.put("user", getUsername());
           usuariosub.put("password", getPassword());
           JSONArray json_usuario_=new JSONArray();
           json_usuario_.put(usuariosub);
           finale.put("usuario",json_usuario_);
       }catch(Exception ex){
           ex.printStackTrace();

        }

        return finale;
    }

	public String getUsername() {
		return username;
	}

	public void setUsername(String username) {
		this.username = username;
	}

	public String getPassword() {
		return password;
	}

	public void setPassword(String password) {
		this.password = password;
	}
}

File: classes/en/Usuario.php
<?php
class EN_Usuario{
    var $idUsuario;
    var $noUsuario;
    var $a1Usuario;
    var $a2Usuario;
    var $feNacimie;
    var $seUsuario;
	var $user;
	var $password;

function setIdUsuario($idUsuario) { $this->idUsuario = $idUsuario; }
function getIdUsuario() { return $this->idUsuario; }
function setNoUsuario($noUsuario) { $this->noUsuario = $noUsuario; }
function getNoUsuario() { return $this->noUsuario; }
function setA1Usuario($a1Usuario) { $this->a1Usuario = $a1Usuario; }
function getA1Usuario() { return $this->a1Usuario; }
function setA2Usuario($a2Usuario) { $this->a2Usuario = $a2Usuario; }
function getA2Usuario() { return $this->a2Usuario; }
function setFeNacimie($feNacimie) { $this->feNacimie = $feNacimie; }
function getFeNacimie() { return $this->feNacimie; }
function setSeUsuario($seUsuario) { $this->seUsuario = $seUsuario; }
function getSeUsuario() { return $this->seUsuario; }
	
function setUser($user) { $this->user = $user; }
function getUser() { return $this->user; }
	
function setPassword($password) { $this->password = $password; }
function getPassword() { return $this->password; }	
	
function generarUsuario($json){
	  if(isset($json)&&!empty($json)){
         
             $data = json_decode($json,true);
         
            $user=$data;
            
            $this->setIdUsuario($user['idusuario']);
            $this->setNoUsuario($user['nousuario']);
            $this->setA1Usuario($user['a1usuario']);
            $this->setA2Usuario($user['a2usuario']);
            $this->setFeNacimie($user['fenacimiento']);
            $this->setSeUsuario($user['sexo']);
		    $this->setUser($user['user']);
            $this->setPassword($user['password']);
           
	  }
	
}
}

// getIdUsuario getNoUsuario getA1Usuario getA2Usuario getFeNacimie getSeUsuario getUser getPassword
?>